eZ/Publish/API/Repository/Repository.php

```diff
@@ -19,6 +19,8 @@
 interface Repository
 {
     /**
+     * @deprecated since 6.5, to be removed. Use PermissionResolver::getCurrentUserReference() instead.
```

Comment:
Updated in https://github.com/ezsystems/ezpublish-kernel/pull/1720/commits/b36e667862234896be49c144d5d43b82dbb7d9b7